Mirror/Runtime/NetworkServer.cs

```diff
@@ -113,6 +113,8 @@ internal static bool InternalListen(string ipAddress, int serverPort, int maxCon
         {
             Initialize();
 
+            connections.Clear();
```

Comment:
Should this go inside Initialize?

Reply:
I did think the same, but Initialize felt like it was a one time method as the 1st thing it does is:
```
if (s_Initialized)
                return;
```
We could put the clear before that, but seems wrong?

Thats why I didnt put it there

Reply:
FTR as discussed in Discord, it would be fine in Initialize so I will move it there in a bit

Reply:
updated as requested @paulpach